model/task/task.go

```diff
@@ -3516,3 +3520,187 @@ func (t *Task) FindAbortingAndResettingDependencies() ([]Task, error) {
 	})
 	return FindAll(q)
 }
+
+// MoveIssueToSuspectedIssue removes an issue from an existing annotation and adds it to its suspected issues,
```

Comment:
Could we have a new file called task/annotations.go or something, just for organization?

Reply:
Done